### ISSUE TITLE
`wp post get $id` returns no `post_title` when the content contains English, Japanese, and Thai language characters

### ISSUE THREAD

**Issue by ShinichiNishikawa** [wp-cli/php-cli-tools#117]
Here's the post_content I have in the editor.

```
♫ มีอีกหลายต่อหลายคน เขาอดทนก็เพื่อรัก
รักผลักดันให้รู้จัก ให้หาหนทางใหม่
ฉันจะล้มตั้งหลายที ดีที่รักมาฉุดไว้
รักสร้างสรรค์สิ่งมากมาย และหลอมละลายทุกหัวใจ
จะมาร้ายดียังไง แต่ใจก็ยังต้องการ
ในทุกๆ วัน โลกหมุนด้วยความรัก ♫
ขอแสดงความยินดี งานแต่งพี่ Earn & Menn
-----
...เที่ยวปายหน้าร้อน ก็เที่ยวได้เหมือนกันน่ะ

ジョバンニはまっ赤になってうなずきました。けれどもいつかジョバンニの眼のなかには涙がいっぱいになりました。そうだ僕は知っていたのだ、もちろんカムパネルラも知っている。

Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore
```

And I attache the output in my terminal.

<img width="1109" alt="2017-08-02 12 08 48" src="https://user-images.githubusercontent.com/1124652/28856140-3bd5a648-777c-11e7-881b-4ec37d52c1f0.png">

**Comment by miya0001** [comment]
Thai language has some chars neither single-width nor double-width, for example "กั" is single-width but 2 char. 😓 
It looks very difficult to solve the problem.

But I found a problem with only Japanese.

I will write a test to represent it.

**Comment by gitlost** [comment]
@ShinichiNishikawa great test case thanks!

@miya0001 
>It looks very difficult to solve the problem.

We can fix it!

**Comment by miya0001** [comment]
Fixed at #118. Thanks @ShinichiNishikawa!
